### PR TITLE
Added maven configuration for DocService

### PR DESCRIPTION
**Pull request by themnd**
Added protobuf-maven-plugin configuration needed to expose services documentation to DocService.

### CODE REVIEW

File: site/src/sphinx/server-docservice.rst [hunk 1]
```diff
@@ -175,6 +175,8 @@ For gRPC:
 - Configure the protobuf plugin to generate the ``.dsc`` file that contains the docstrings and
   put it into the ``META-INF/armeria/grpc`` directory:
 
+  Using gradle:
```
Comment (trustin) [inline site/src/sphinx/server-docservice.rst]: gradle -> Gradle

File: site/src/sphinx/server-docservice.rst [hunk 2]
```diff
@@ -188,3 +190,31 @@ For gRPC:
               }
           }
       }
+
+  Using maven:
```
Comment (trustin) [inline site/src/sphinx/server-docservice.rst]: maven -> Maven

File: site/src/sphinx/server-docservice.rst [hunk 3]
```diff
@@ -188,3 +190,31 @@ For gRPC:
...
+
+  .. code-block:: xml
+
+    <plugin>
```
Comment (trustin) [inline site/src/sphinx/server-docservice.rst]: How about adding some comment block to the full manual of `protobuf-maven-plugin`? e.g.

```xml
<!-- See https://www.xolstice.org/protobuf-maven-plugin/usage.html for more information. -->
```

**Comment by themnd** [comment]
Incorporated your observations.

**Comment by trustin** [comment]
:+1:

**Pull request merged**